src/pcapi/core/offers/repository.py

```diff
@@ -231,10 +231,10 @@ def get_stocks_by_id_at_providers(id_at_providers: List[str]) -> Dict:
         Stock.query.filter(Stock.idAtProviders.in_(id_at_providers))
         .outerjoin(Booking, and_(Stock.id == Booking.stockId, Booking.isCancelled.is_(False)))
         .group_by(Stock.id)
-        .with_entities(Stock.id, Stock.idAtProviders, coalesce(func.sum(Booking.quantity), 0))
+        .with_entities(Stock.id, Stock.idAtProviders, coalesce(func.sum(Booking.quantity), 0), Stock.offerId)
         .all()
     )
     return {
-        id_at_providers: {"id": id, "booking_quantity": booking_quantity}
-        for (id, id_at_providers, booking_quantity) in stocks
+        id_at_providers: {"id": id, "booking_quantity": booking_quantity, "offerId": offerId}
+        for (id, id_at_providers, booking_quantity, offerId) in stocks
```

Comment:
```suggestion
        id_at_providers: {"id": id, "booking_quantity": booking_quantity, "offer_id": offer_id}
        for (id, id_at_providers, booking_quantity, offer_id) in stocks
```

parce que j'aime bien chipoter ;)